### DISCUSSION TITLE
SEB Server has no option to specify Start URL under "General" tab

### DISCUSSION THREAD

**Discussion by imrhlrvndrn** [SafeExamBrowser/seb-server#101] [Q&A] [answered]
[@anhefti](https://github.com/anhefti) With your valuable inputs the setup was successful. As an EdTech company, we have our own custom learning management system where we administer all kinds of exams. It is now our intention to build an exam system that will be more serious in nature, and, therefore, a service such as SEB is our preferred choice.

## Current exam flow
1. The SEB Client is used for exams where the start URL is a Google Sheet with questions. Students make a copy of that file and work on their solution.

1. As part of the same question sheet, we provide a Google Form link where the student can submit his/her details and a Google Sheet link at the end of the exam.

## Challenges we’re facing

1. Currently, we do not have a way to retrieve information about the ongoing sessions.

1. This means we cannot determine whether a student has successfully completed the exam on SEB Client or whether they have copied the question file & submission form and terminated the session.

    1. Which would allow them to work on their solution using external resources that SEB blocks

1. We were therefore interested in SEB Servers Exam Monitoring.

Can the current flow be continued with the Start URL as a Google Sheet link? We were not able to specify the Start URL in the SEB Server `Exam Configuration > View Exam Configuration > Edit SEB Settings > Tab (General)` which we used to define the entrypoint in the SEB client

The `.seb` file generated via `Exam Configuration > View Exam Configuration > Edit SEB Settings > Export SEB Settings` when loaded into a SEB Client takes us to the default SEB page [safeexambrowser.org/start](https://safeexambrowser.org/start)

How can we connect the SEB Client with the SEB Server which runs locally for testing out the functionalities. Also is _**Exam Monitoring**_ available only  for Exams created via LMS Setup?

![Seb Server Exam Config](https://github.com/SafeExamBrowser/seb-server/assets/43089715/f2a45d19-c5ae-4284-b5f1-803a648fd36d)

**Comment by anhefti** [comment, marked as answer]
Hi,

Thank you for using SEB Server.
Glad to hear you were successful on setting up SEB Server. Unfortunately, your described use case is yet not supported by SEB Server since SEB Server currently allows only secure integrations with supported learning management systems (LMS) such as Moodle, OpenEdx, Ans, and OpenOLAT. But we will support exams with SEB Server and simple webpages as you described within your use-case, with the next oncoming SEB Server version 1.6 (in Q2 this year).

With the new features, your use-case would be possible to apply with SEB and SEB Server. 

> Can the current flow be continued with the Start URL as a Google Sheet link? We were not able to specify the Start URL in the SEB Server Exam Configuration > View Exam Configuration > Edit SEB Settings > Tab (General) which we used to define the entrypoint in the SEB client
> 
> The .seb file generated via Exam Configuration > View Exam Configuration > Edit SEB Settings > Export SEB Settings when loaded into a SEB Client takes us to the default SEB page [safeexambrowser.org/start](https://safeexambrowser.org/start)
> 
> How can we connect the SEB Client with the SEB Server which runs locally for testing out the functionalities. Also is Exam Monitoring available only for Exams created via LMS Setup?

In SEB Server the process to connect SEB with SEB Server is a little different than you assumed above. Therefore, you need to download the Connection Configuration either directly or within the exam setup page:

Directly: Connection Configuration > View Connection Configuration > Export Connection Configuration
Within Exam: Exam > View Exam > Export Exam Connection Configuration

With both cases you get a .seb File that can be loaded with SEB and SEB will securely connect to SEB Server first. In the first case, SEB will present all Exams currently running on SEB Server to the user for choice. In the second case, the exam is already preselected and the User gets directly to the start page of the exam after the prepared Exam Configuration has been downloaded and reconfigured by the SEB automatically. Exporting Exam Configuration directly is not a real use-case, but more for testing purposes. SEB should always start with a Connection Configuration when working with SEB Server. 

[Please also read the SEB Server User Manual.](https://seb-server.readthedocs.io/en/latest/#)

If you are interested using SEB Server in production and would like to apply for more support in the future, please consider joining the [SEB Alliance](https://www.safeexambrowser.org/alliance/members.html).

Best regards
